styles/custom.scss

```diff
@@ -3,6 +3,30 @@ $theme-colors: (
   "secondary": #00bfa5,
 );
 
+$link-color: #0d3e79;
+$hr-color: #dee2e6;
+$hr-opacity: 1;
+$link-decoration: none;
```

Comment:
here i am overriding some of the default variables to get back to our previous design + our links are styled the same throughout the website so this will take less code and keep us consistent